src/filterConfig.js

```diff
@@ -4,6 +4,21 @@ import {
   itemFilterRenderer,
 } from './components';
 
+// Function which takes a filter name and returns
+// another function which can be used in filter config
+// to parse a given filter into a CQL manually.
+const parseFilter = name => values => {
```

Comment:
Ooooh! Thank you for the comments!

How about `generateOptionalBoolean` instead of `parseFilter`? Parse is quite generic, more in line to what I'd name the filter-handler in stripes-components. But here, there might be `generateOptionalBooleanFilter` or `buildDateRangeFilter` etc.

Reply:
@zburke I completely missed this comment but I agree. We should rename it!